src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -33,7 +33,7 @@ export function getDummyNamespace(name: string) {
 export class NamespaceStore extends KubeObjectStore<Namespace> {
   api = namespacesApi;
 
-  @observable private contextNs = observable.set<string>();
+  @observable private rawSelectedNamespaces = observable.set<string>();
```

Comment:
`contextNS` used to be public property. After changing it private extensions developer could not use it anymore. However current extensions still did not break. This change will make also current extensions to break if they are relying on this property.

Reply:
It maybe have been previously `public` but it currently isn't.

Reply:
Yes, it's definitely private now, but it's causing build time failure now for extensions developers, existing lens extension instances are still running fine because they are javascript. 

I'm just saying that if we are going to include this PR in 4.1.3 it will then definitely break those extensions that are already installed in Lens. 

For example used here: https://github.com/nevalla/lens-resource-map-extension/blob/master/components/NamespaceSelect.tsx#L11

Reply:
Ah of course, extensions can ignore our visibility modifiers. Given that `NamespaceSelect` is now public I assume you should use that?

Reply:
I can do a follow-up PR to target this and not to introduce breaking change.

Reply:
@Nokel81 any reason why we cannot make this PR so that it does not break extensions (runtime)? If not -> please change this to be compatible with existing extensions.

Reply:
could be something like this:
```typescript
/** 
* @depreated use NamespaceStore.selectedNamespaces instead 
*/
get contextNs() {
    return this.selectedNamespaces;
}
```

Reply:
That is what I was planning on doing @nevalla